docs/freqtrade-101.md

```diff
@@ -0,0 +1,106 @@
+
+For simplicity, here we will focus on spot (long) trades only.
+
+## Structure of a Basic Strategy
+
+### Main dataframe
+
+Freqtrade strategies use a tabular data structure with rows and columns known as a `dataframe` to generate signals to enter and exit trades.
+
+Freqtrade dataframes are organised by pair, and are indexed by the `date` column, e.g. `2024-06-31 12:00`.
+
+The next 5 columns represent the `open`, `high`, `low`, `close` and `volume` (OHLCV) data.
+
+### Populate indicator values
+The `populate_indicators` function adds columns to the dataframe that represent the technical analysis indicator values.
+
+Examples of common indicators include Relative Strength Index, Bollinger Bands, Money Flow Index, Moving Average, and Average True Range.
+
+Columns are added to the dataframe by calling technical analysis functions, e.g. ta-lib's RSI function `ta.RSI()`, and assigning them to a column name, e.g. `rsi`
+
+```python
+    dataframe['rsi'] = ta.RSI(dataframe)
+```
+
+### Populate entry signals
+The `populate_entry_trend` function defines conditions for an entry signal.
+
+The dataframe column `enter_long` is added to the dataframe, and when a value of `1` is in this column, Freqtrade sees an entry signal.
+
+??? Hint "Shorting"
+    When shorting, this column is called `enter_short`.
+
+### Populate exit signals
+The `populate_exit_trend` function defines conditions for an exit signal.
+
+The dataframe column `exit_long` is added to the dataframe, and when a value of `1` is in this column, Freqtrade sees an exit signal.
+
+??? Hint "Shorting"
+    When shorting, this column is called `exit_short`.
+
+## A simple strategy
+
+Here is a minimal example of a Freqtrade strategy:
+
+```python
+from freqtrade.strategy.interface import IStrategy
+from pandas import DataFrame
+
+class MyStrategy(IStrategy):
+
+    def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+        # generate values for technical analysis indicators
+        dataframe['rsi'] = ta.RSI(dataframe, timeperiod=14)
+
+        return dataframe
+
+    def populate_entry_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+        # generate entry signals based on indicator values
+        dataframe.loc[
+            (dataframe['rsi'] < 30),
+            'enter_long'] = 1
+
+        return dataframe
+
+    def populate_exit_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+        # generate exit signals based on indicator values
+        dataframe.loc[
+            (dataframe['rsi'] > 70),
+            'exit_long'] = 1
+
+        return dataframe
+```
+
+## Making trades
+When a signal is found (a `1` in an entry or exit column), Freqtrade will attempt to make an order on the exchange, i.e. a `trade`.
```

Comment:
I'd extend this a bit mentioning that `signal != order` ... as that's far from a given.

Reply:
OK good point. I'll add that.